src/Components/ProgressBar/ProgressBar.scss

```diff
@@ -25,7 +25,7 @@
 .cf-progress-bar--text {
   margin-top: $cf-space-2xs;
   width: 100%;
-  color: $cf-grey-55;
+  color: #88889b;
```

Comment:
Will replace this in a future PR with a variable. That color is used as a hex in some more places.